coreMQTT/build.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -e
+
+COREMQTT_VERSION="2.3.0"
+COREMQTT="coreMQTT-${COREMQTT_VERSION}"
+PKG_URL="https://github.com/FreeRTOS/coreMQTT/archive/refs/tags/"
+
+b_log "Building coreMQTT"
+PREFIX_COREMQTT="${PREFIX_PROJECT}/phoenix-rtos-ports/coreMQTT"
+PREFIX_COREMQTT_BUILD="${PREFIX_BUILD}/coreMQTT"
+PREFIX_COREMQTT_SRC="${PREFIX_COREMQTT_BUILD}/${COREMQTT}/source"
+PREFIX_COREMQTT_INSTALL="${PREFIX_COREMQTT_BUILD}/install"
```

Comment:
phoenix-rtos-ports already provides `PREFIX_PORT` and `PREFIX_PORT_BUILD` (which is also created) so you may use these instead of `PREFIX_COREMQTT` and `PREFIX_COREMQTT_BUILD`

Reply:
Done